frontend/src/modules/projects/board.tsx

```diff
@@ -109,34 +123,52 @@ export default function Board() {
     );
   }, [submenuItemsOrder[workspace.id]]);
 
+  useEffect(() => {
```

Comment:
This could be abstracted away in a custom hook to make the code a little more clear.